spaces/models.py

```diff
@@ -0,0 +1,23 @@
+from django.db import models
+from django.contrib.auth.models import User
+
+
+class Address(models.Model):
+    street_name = models.CharField(max_length=140)
+    civic = models.CharField(max_length=20)
```

Comment:
Does that even exist outside Canada?

ON the contrary, I think it's too specific. I would just say

```
address_line1 = models.CharField(max_length=70)
address_line2 = models.CharField(max_length=70)
# address_line3 (maybe)
```